zerver/webhooks/papertrail/fixtures/long_post.json

```diff
@@ -11,7 +11,7 @@
       "program": "CROND",
       "severity": "Info",
       "facility": "Cron",
-      "message": "message body 1",
+      "message": "message body 1"
```

Comment:
Papertrail isn’t really going to send us invalid JSON…right?

Reply:
The [examples](https://help.papertrailapp.com/kb/how-it-works/search-api/#examples) in the Papertrail documentation look valid.

Reply:
Seems like we should merge this under the assumption it's a hand-entry error of some sort.